tokio-signal/src/unix.rs

```diff
@@ -84,6 +62,131 @@ impl Init for OsExtraData {
     }
 }
 
+/// Represents the specific kind of signal to listen for.
+#[derive(Debug, Clone, Copy)]
+pub struct SignalKind(c_int);
```

Comment:
At a bikeshed level, `SignalKind` isn't my favorite name... but it does work. It has precedence w/ `io::ErrorKind` and I can't think of anything better :).